OData/test/System.Web.OData.Test/OData/OpenEntityTypeTest.cs

```diff
@@ -470,33 +447,108 @@ public IHttpActionResult Put(int key, SimpleOpenCustomer changedCustomer)
             Assert.Equal(1, changedCustomer.CustomerProperties.Count);
             return Updated(changedCustomer); // Updated(customer);
         }
+
+        private static IList<SimpleOpenCustomer> GetCustomers()
```

Comment:
Use the existing method name: CreateCustomers, which could allow view the diff easier.